generators/app/templates/.storybook/head.html

```diff
@@ -2,5 +2,8 @@
 <script>
   var root = document.getElementsByTagName( 'html' )[0];
   root.setAttribute( 'class', 'terra-Application' );
-  root.setAttribute( 'dir', 'ltr' );
+  document.addEventListener( 'DOMContentLoaded', function() {
```

Comment:
NIT: if this script was loaded at the end of the body you could skip the event listener.